manifests/crd/odhapplications.dashboard.opendatahub.io.crd.yaml

```diff
@@ -43,6 +43,10 @@ spec:
                   type: string
                 category:
                   type: string
+                  enum:
```

Comment:
Undo this based on the conversation in slack from a few days ago. I'd like to see this make this sprint and they are hesitant about CRD changes due to CRs in flight out there